src/client/renderer/r_types.h

```diff
@@ -398,7 +397,6 @@ typedef struct r_material_s {
 	r_image_t *tintmap;
 
 	uint32_t time;
-	uint32_t flags; // these may differ from cm->flags
```

Comment:
I was wondering about this too, my assumption was that this existed so that we can test if a material has x stage flag/is diffusable/etc instead of having to sort through all of the stages to figure that out. I'll have to read into it further but I'm wondering how you got the STAGE_DIFFUSE/LIGHTING, etc working if this is gone now.

EDIT: ok, I see now that it's because cm->flags is used instead now. Good idea, this does seem a bit redundant.